packages/components/src/Link/Link.tsx

```diff
@@ -20,12 +20,15 @@ export type LinkProps = LinkHTMLElementProps & {
 /**
  * Component for making anchor tags.
  *
- * This component is called Link because it should be used to making link elements;
- * however, it can be styled to look like a button. In terms of the look and feel of the
- * component in the UI, the Link and Button components are exactly the same.
+ * This component is called Link because it should be used to make link elements;
```

Comment:
There's some pedantic nerd deep inside me somewhere that wants to update this to

```suggestion
 * This component is called Link because it should be used to make anchor elements;
```

But it doesn't really matter... 🤔  No, doesn't really matter

Reply:
No, I think that's a good point – I can see "link element" being confusing. I'll change it to `<a>` to be consistent with the rest of the text. Thanks!